CHANGES.md

```diff
@@ -7,6 +7,10 @@ Change Log
 
 * Fixed a bug causing custom TilingScheme classes to not be able to use a GeographicProjection. [#6524](https://github.com/AnalyticalGraphicsInc/cesium/pull/6524)
 
+##### Additions :tada:
+
+* Added support for loading Draco encoded Point Cloud tiles [#6559](https://github.com/AnalyticalGraphicsInc/cesium/pull/6559)
```

Comment:
Add a tad more detail, e.g., "compressed", not "encoded", and something like "up to 10x" or whatever compression.